src/components/ScrollableView/index.tsx

```diff
@@ -46,7 +47,7 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-export default function ScrollableView(props) {
+export default function ScrollableView(props: PropsWithChildren) {
     const { children } = props;
```

Comment:
```suggestion
export default function ScrollableView({ children }: PropsWithChildren) {
```
Not a big difference, but it's a bit cleaner